packages/fxa-auth-server/lib/senders/emails/sass-compile-files.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { renderSync } from 'sass';
+import {
+  writeFileSync,
+  mkdirSync,
+  existsSync,
+  rmdirSync,
+  readdirSync,
+} from 'fs';
+import path from 'path';
+
+const getDirectories = (source: string) =>
+  readdirSync(source, { withFileTypes: true })
+    .filter((dirent: any) => dirent.isDirectory())
+    .map((dirent: any) => dirent.name);
+
+// construct arrays of partials and templates based on the directories
+const partials = getDirectories(path.join(__dirname, `partials`));
```

Comment:
This is a tiny nitpick, but we only really need the backtick if there's a variable in the string. We can change a lot of thes in this file beginning with backticks to `'`.